### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Sourolio10**
I have added Lungs v2 results for Azimuth vs ASCTB comaparisons

### CODE REVIEW

File: pages/asctb-azimuth-data-comparison.html [hunk 1]
```diff
@@ -52,8 +52,7 @@
                     <a class="dropdown-item" href="https://portal.hubmapconsortium.org/ccf-eui" title="Opens external website" target="_blank">CCF Exploration User Interface (EUI)</a>
                     <a class="dropdown-item" href="https://expand.iu.edu/browse/sice/cns/courses/hubmap-visible-human-mooc" title="Opens external website" target="_blank">Visible Human MOOC</a>
                     <a class="dropdown-item" href="https://portal.hubmapconsortium.org/" title="Opens external website" target="_blank">HuBMAP Data Portal</a>
-                    <a class="dropdown-item" href="kaggle.html" title="Opens external website" target="_blank">Kaggle #1: HuBMAP - Hacking the Kidney</a>
-                    <a class="dropdown-item" href="kaggle2.html" title="Opens external website">Kaggle #2: HuBMAP + HPA - Hacking the Human Body</a>
+                    <a class="dropdown-item" href="kaggle.html" title="Opens external website" target="_blank">Kaggle Competition and Awards</a>
```
Comment (bherr2) [inline pages/asctb-azimuth-data-comparison.html]: Revert this

File: pages/asctb-azimuth-data-comparison.html [hunk 2]
```diff
@@ -102,7 +101,17 @@ <h3 class="mt-5">Comparison of CxG in ASCT+B vs. Azimuth</h3>
                         <p>Table 3 shows the comparison of the CxG matrix in ASCT+B tables vs. CxG matrix in Azimuth for each organ in Jan 2022. An automatically generated, up-to-date comparison is available <a href="https://hubmapconsortium.github.io/asctb-azimuth-data-comparison/Azimuth_vs_ASCTB.summaries.xlsx">here</a>; this xls also contains separate sheets with cell types that are not (yet) covered in the ASCT+B tables. Details on how the comparison is calculated are on <a href="https://github.com/hubmapconsortium/asctb-azimuth-data-comparison">GitHub</a>. Note that not all Azimuth cell types have CL IDs; in these cases, matching is done based on cell type names.</p>
 
                         <p>Data from <a href="https://github.com/satijalab/azimuth_website" title="Satija Labs Azimuth website">https://github.com/satijalab/azimuth_website</a> has been parsed to derive cell type typologies:</p>
+                      <!-- <h2 class="mt-5">Download Data</h2> -->
```
Comment (bherr2) [inline pages/asctb-azimuth-data-comparison.html]: Remove commented out code

File: pages/asctb-azimuth-data-comparison.html [hunk 3]
```diff
@@ -156,18 +166,20 @@ <h3 class="mt-5">Comparison of CxG in ASCT+B vs. Azimuth</h3>
...
+                                            <td class="text-right">N/A</td>
                                             <td class="text-right">4</td>
                                             <td class="text-right">24,402</td>
                                             <td class="text-right">864</td>
                                             <td class="text-right">254</td>
                                             <td class="text-right">152</td>
                                         </tr>
                                         <tr>
-                                            <td class="align-top"><a href="https://github.com/hubmapconsortium/asctb-azimuth-data-comparison/blob/gh-pages/Heart.csv" title="Heart csv file">Heart</a></td>
+                                            <td class="align-top"><a href="https://github.com/hubmapconsortium/asctb-azimuth-data-comparison/blob/gh-pages/Heart.csv" title = "Heart csv file">Heart</a></td>
```
Comment (bherr2) [inline pages/asctb-azimuth-data-comparison.html]: remove spaces added here.. `title = "` => `title="`

File: pages/asctb-azimuth-data-comparison.html [hunk 4]
```diff
@@ -208,12 +222,41 @@ <h3 class="mt-5">Comparison of CxG in ASCT+B vs. Azimuth</h3>
...
+                                            <td class="text-right">N/A</td>
+                                            <td class="text-right">4</td>
+                                            <td class="text-right">9</td>
+                                            <td class="text-right">5,104</td>
+                                            <td class="text-right">N/A</td>
+                                            <td class="text-right">N/A</td>
+                                            <td class="text-right">N/A</td>
+                                        </tr>
+                                        <tr>
+                                            <td class="align-top"><a href="https://github.com/hubmapconsortium/asctb-azimuth-data-comparison/blob/gh-pages/Human%20Lung%20v2.celltype_stats.csv" title="Lung v2 csv file">Lung v2</a></td>
```
Comment (bherr2) [inline pages/asctb-azimuth-data-comparison.html]: Use the other url pattern here:
<https://hubmapconsortium.github.io/asctb-azimuth-data-comparison/Human%20Lung%20v2.celltype_stats.csv>

Also, it'd be better to rename to remove uppercase letters and replace space with dash.

**Review by bherr2 — CHANGES_REQUESTED** [review]
@Sourolio10 some changes needed

**Comment by bherr2** [comment]
@Sourolio10 it's been seven days. Can you address my PR comments please?

File: pages/asctb-azimuth-data-comparison.html (hunk 1 above)
Comment (Sourolio10) [reply pages/asctb-azimuth-data-comparison.html]: Exactly what should I revert here, Bruce?

Comment (bherr2) [reply pages/asctb-azimuth-data-comparison.html]: Remove the green line and re-add the red lines

File: pages/asctb-azimuth-data-comparison.html (hunk 4 above)
Comment (bherr2) [reply pages/asctb-azimuth-data-comparison.html]: @Sourolio10 Use the other url pattern here:
https://hubmapconsortium.github.io/asctb-azimuth-data-comparison/Human%20Lung%20v2.celltype_stats.csv

Also, it'd be better to rename to remove uppercase letters and replace space with dash.

**Review by bherr2 — APPROVED** [review]

**Pull request merged**